webapi/src/main/scala/org/knora/webapi/http/handler/KnoraExceptionHandler.scala

```diff
@@ -74,6 +75,21 @@ object KnoraExceptionHandler extends LazyLogging {
         }
       }
 
+    case bre: V2.BadRequestException =>
```

Comment:
Could we call it `badRequestException` instead of `bre`?

Reply:
I followed convention applied in other cases, which means no :)